cmd/server.go

```diff
@@ -185,6 +208,10 @@ func init() {
 	serverCreateCmd.PersistentFlags().StringVar(&hostName, "hostname", "", "Hostname")
 	serverCreateCmd.PersistentFlags().StringVar(&plainPassword, "password", "", "Plain-text password")
 
-	serverCmd.AddCommand(serverLsCmd, serverOnCmd, serverOffCmd, serverRmCmd, serverCreateCmd)
+	serverSetCmd.PersistentFlags().IntVar(&memory, "mem", 1, "Memory (GB)")
+	serverSetCmd.PersistentFlags().IntVar(&cores, "cores", 1, "No. of cores")
+	serverSetCmd.PersistentFlags().StringVar(&serverName, "name", "", "Name of the server")
```

Comment:
You give here default values. Suppose I have a 1 core, 2G mem machine, doesn't this mean that

    $ gscloud server set <ID> --cores 2

will also reduce memory from 2 to 1?

Reply:
Ah, i see 🤭

Reply:
Fixed by using 0, now i can upgrade even when the server is running, thanks for the hint 👍